tests/components/google_assistant_sdk/test_init.py

```diff
@@ -180,6 +189,78 @@ async def test_send_text_command_expired_token_refresh_failure(
     assert any(entry.async_get_active_flows(hass, {"reauth"})) == requires_reauth
 
 
+async def test_send_text_command_media_player(
```

Comment:
The expiration also seems like an important thing to test. Typically this is done with `async_fire_time_changed `.  Speculation:... though you may need to use `async_call_later` or one of the home assistant variants to make this happen, rather than calling call later on the loop directly.

Reply:
Done. 
I copied this call later on the loop directly from tts, where it's untested. I wanted to test it but I couldn't figure out how. Thanks for pointing me to the right functions!